llama_stack/apis/scoring/scoring.py

```diff
@@ -48,11 +48,13 @@ class Scoring(Protocol):
     async def score_batch(
         self,
         dataset_id: str,
-        scoring_functions: List[str],
+        scoring_functions: Optional[Dict[str, ScoringFnParams]] = None,
```

Comment:
Wrong Optional here. It should be

```
Dict[str, Optional[ScoringFnParams]]
```

you have to pass a non-empty Dict otherwise what are you scoring?